.github/workflows/ci.yml

```diff
@@ -73,6 +73,8 @@ jobs:
           platforms: arm64
       - name: Test Docker Image
         run: core/docker/build.sh
+      - name: Block Merge Commits
+        uses: nineinchnick/block-commits-action@cea2eb9b1b83c500d877cd60649d98b58fc2423c
```

Comment:
Can we put this somewhere under our github org?
Do we need a new repo, for actions?

Reply:
In #12817 I have a few actions that I added to `nineinchnick/github-actions` and I suggested to transfer this repo to the `trinodb` org. If you'd create it, I could add this action there too.